src/modules/esl-event-listener/core/targets/resize.adapter.ts

```diff
@@ -1,47 +1,54 @@
 import {SyntheticEventTarget} from '../../../esl-utils/dom/events/target';
 import {overrideEvent} from '../../../esl-utils/dom/events/misc';
 
+import type {ESLDomElementTarget} from '../../../esl-utils/abstract/dom-target';
+
+export type ESLResizeTarget = ESLDomElementTarget | Window;
```

Comment:
Consider removing it due to only one actual usage, and no practical need for users as well